samples/bluetooth/mesh/sensor_server/sample.yaml

```
@@ -0,0 +1,9 @@
+sample:
+  name: Bluetooth Mesh Sensor Server
+  description: Bluetooth Mesh Sensor Server sample
```

```suggestion
  name: Bluetooth Mesh sensor server
  description: Bluetooth Mesh sensor server sample
```
Not sure if this is visible to the customer. If not, then I guess it is not that important if these are capitalized either.